documentation/man/features/mail.rst

```diff
@@ -20,15 +20,28 @@ It is common to deal with patch submissions to a mailing list, and
 **mail** functionality aims to wrap this tool to simplify its usage and
 integrate it with other **kw** functionalities.
 
+.. warning::
+  If any *<extra-args>* start with a ``-`` they **need** to be placed after the
+  ``--`` argument.
+
 OPTIONS
 =======
 -s, \--send:
   Send an email using ``git send-email`` to the email adresses specified with
-  ``--to``. This sends the last commit.
+  ``--to``. You can provide *<extra-args>* to be passed directly to
+  ``git send-email``, if they start with a ``-`` they **need** to come after
+  the ``--`` argument.
```

Comment:
Try to expand this documentation a little bit more; in particular, try to describe the `--` behavior a little bit better. I recommend you add a paragraph that describes this send mail feature's core idea and add examples.